haiku-apps/helpviewer/helpviewer-1.6.2.1.recipe

```diff
@@ -6,8 +6,8 @@ HOMEPAGE="http://fatelk.com/"
 COPYRIGHT="2017-2019 Jim Saxton, Fat Elk Software"
 REVISION="1"
 LICENSE="FatElk"
-SOURCE_URI="http://fatelk.com/helpviewer/helpviewer-1.6.1.2.zip"
-CHECKSUM_SHA256="c00e18a2f666ec9e1a354fde5151fba0e9dc32a063265399c1e1e39bc0943a39"
+SOURCE_URI="http://fatelk.com/helpviewer/helpviewer-1.6.2.1.zip"
```

Comment:
I'm not sure I understand why it would require multiple zip files.  Inside of this recipe $portVersion will equal 1.6.2.1 and be effectively the same thing.  So the only difference is that you won't have to change the SOURCE_URI every time you do a new release in the future because it will get that information from the recipe file name.

Reply:
ok, sounds good.